samples/app-anonymous-users/csharp/ClientApp/src/components/shareview.jsx

```diff
@@ -19,11 +20,30 @@ const ShareView = () => {
     const [uShowCount, uShowSetCount] = useState(0);
 
     // Declare new state variables that are required to disable the submit vote button.
-    const [disabled, setDisabled] = useState(false)
+    const [disabled, setDisabled] = useState(false);
+
+    // Declare new state variables that are required to assgin facebook user name
```

Comment:
This comment looks wrong, as it seemed it's used to set the name of users who sign in using SSO too

Reply:
Updated your feedback.